packages/lodestar/src/network/reqresp/reqUtils.ts

```diff
@@ -87,10 +87,10 @@ export async function* sendRequestStream<T extends ResponseBody>(
   logger.verbose("sent request", {peer: peerId.toB58String(), method});
 
   yield* pipe(
-    abortDuplex(conn!.stream, controller.signal, {returnOnAbort: true}),
+    (abortDuplex(conn!.stream, controller.signal, {returnOnAbort: true}) as unknown) as AsyncIterable<Buffer>,
```

Comment:
maybe we can add better types for abort duplex?

Reply:
Agree, but would you mind opening an issue for it? I think it's out the scope of this PR

Reply:
But if you don't fix it, isn't it basically same thing as before(if something changes it will happily cast it into type that fits)? 😄

Reply:
I'm not adding type unsafety, this change just shows what was already there. The goal of the PR is to allow future work to be type safe. Same thing when turning strictNullChecks on, the goal of the work was not to fix existing issues but to allow new code to work with the rule